dotcom-rendering/src/components/Card/components/CardWrapper.tsx

```diff
@@ -134,17 +133,11 @@ const onwardContentHoverStyles = css`
 	}
 `;
 
-const topBarStyles = ({
-	isDynamo,
-	format,
-}: {
-	isDynamo?: true;
-	format: ArticleFormat;
-}) => {
+const topBarStyles = ({ isDynamo }: { isDynamo?: true }) => {
 	/* Styling for top bar */
 	const baseStyles = css`
 		background-color: ${isDynamo
-			? decidePalette(format).text.dynamoKicker
+			? palette('--card-kicker-text')
```

Comment:
I don't think this is correct, hence you have all these diffs in the dynamo. `text.dynamoKicker` from `decidePalette` was [taking into account the colour from the palettes](https://github.com/guardian/dotcom-rendering/blob/642eef8350e820e32f9e73544627ab4aa2ef154e/dotcom-rendering/src/lib/decidePalette.ts#L865)

Reply:
Actually, the problem was that `text.dynamoKicker` did **NOT** take into account overrides, as you can tell from the old signature not receiving the overrides.

I think the solution here would be to either:
- create a distinct colour in the palette
- make the `ContainerOverrides` only wrap the children, rather that the wrapper itself